### PR TITLE
Feat/add search form e2e

### PR DESCRIPTION
**Pull request by Thistle-Strum**
🔥 Summary
Cypress e2e tests for search-form component
https://github.com/casper-network/casper-blockexplorer-frontend/assets/88854201/7917f526-c07d-47ea-811a-fbde07d5ddf3

### CODE REVIEW

**Review by hoffmannjan — APPROVED** [review]
The tests looking good, I'm just not sure about the config but I bet we can find it out later

File: test/cypress/fixtures/react-select-selectors.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (mgibson-casper) [inline test/cypress/fixtures/react-select-selectors.json:1]: Nice 👍

File: test/cypress/e2e/search-form.cy.ts [hunk 2]
```diff
@@ -0,0 +1,214 @@
+// eslint-disable-next-line import/extensions
+import reactSelectSelectors from '../fixtures/react-select-selectors.json';
+
+const accountHash =
```
Comment (mgibson-casper) [inline test/cypress/e2e/search-form.cy.ts]: Could we put these into another file please?

**Review by mgibson-casper — APPROVED** [review]

File: test/cypress/e2e/search-form.cy.ts (hunk 2 above)
Comment (Thistle-Strum) [reply test/cypress/e2e/search-form.cy.ts]: Relocated!

Comment (Thistle-Strum) [reply test/cypress/e2e/search-form.cy.ts]: Relocated!

**Pull request merged**